src/com/oltpbenchmark/DBWorkload.java

```diff
@@ -907,9 +906,35 @@ private static Results runWorkload(List<BenchmarkModule> benchList, boolean verb
         LOG.info("Num New Order transactions : " + numNewOrderTransactions + ", time seconds: " + time);
         LOG.info("TPM-C: " + tpmc);
         LOG.info("Efficiency : " + efficiency);
+        printLatencies(workers, workConfs.get(0).getTransTypes());
         return r;
     }
 
+    private static void printLatencies(List<Worker<?>> workers, TransactionTypes transactionTypes) {
+        List<List<Integer>> list_latencies = new ArrayList<>();
+        for (int i = 0; i < 5; ++i) {
+            list_latencies.add(new ArrayList<Integer>());
+        }
+        for (Worker<?> w : workers) {
+            for (LatencyRecord.Sample sample : w.getLatencyRecords()) {
+                list_latencies.get(sample.tranType - 1).add(sample.operationLatencyUs);
+            }
+        }
+        for (int i = 0; i < 5; ++i) {
+            Collections.sort(list_latencies.get(i));
```

Comment:
This will be a problem for long running workloads. Please add a TODO to fix this method of calculation.

Reply:
Done.